packages/@aws-cdk/aws-route53/lib/hosted-zone.ts

```diff
@@ -59,6 +59,8 @@ export class HostedZone extends Resource implements IHostedZone {
   /**
    * Import a Route 53 hosted zone defined either outside the CDK, or in a different CDK stack
    *
+   * Use when hosted zone ID is known and hosted zone name is undesirable.
```

Comment:
maybe reword to say that the zone name will be unavailable if this API is used for import